diff --git a/app/navigator/settings/base.py b/app/navigator/settings/base.py
index ae66558..3000b91 100644
--- a/app/navigator/settings/base.py
+++ b/app/navigator/settings/base.py
@@ -9,14 +9,16 @@ https://docs.djangoproject.com/en/1.9/ref/settings/
 """
 
 import os
+import sys
 
 import dj_database_url
 from easy_thumbnails.conf import Settings as thumbnail_settings
 import environ
 
-
 env = environ.Env()
-
+env.read_env('.new_env')
+if 'pytest' in sys.modules or 'test' in sys.argv:
+    env.read_env('.new_env.test')
 
 PROJECT_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
