diff --git a/conf/settings.py b/conf/settings.py
index 2ee024b..8af3384 100644
--- a/conf/settings.py
+++ b/conf/settings.py
@@ -11,6 +11,7 @@ https://docs.djangoproject.com/en/1.9/ref/settings/
 """
 
 import os
+import sys
 
 import environ
 
@@ -18,7 +19,9 @@ from directory_constants import cms
 
 
 env = environ.Env()
-env.read_env()
+env.read_env('.new_env')
+if 'pytest' in sys.modules or 'test' in sys.argv:
+    env.read_env('.new_env.test')
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 
