diff --git a/conf/settings.py b/conf/settings.py
index a46fb71..c540b07 100644
--- a/conf/settings.py
+++ b/conf/settings.py
@@ -11,14 +11,16 @@ https://docs.djangoproject.com/en/1.9/ref/settings/
 """
 
 import os
+import sys
 
 import environ
 
 from directory_constants.constants import cms
 
-
 env = environ.Env()
-env.read_env()
+env.read_env('.new_env')
+if 'pytest' in sys.modules or 'test' in sys.argv:
+    env.read_env('.new_env.test')
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 
