diff --git a/.gitignore b/.gitignore
index 91dcd98..a1db25b 100644
--- a/.gitignore
+++ b/.gitignore
@@ -106,3 +106,7 @@ package-lock.json
 # Docker
 .env
 media
+
+# Database dump
+fixtures/dump.json
+fixtures/dump_last_migration
diff --git a/conf/settings.py b/conf/settings.py
index cfb3b39..d0d6ea3 100644
--- a/conf/settings.py
+++ b/conf/settings.py
@@ -13,13 +13,15 @@ https://docs.djangoproject.com/en/1.9/ref/settings/
 '''
 
 import os
+import sys
 
 import dj_database_url
 import environ
 
-
 env = environ.Env()
-env.read_env()
+env.read_env('.new_env')
+if 'pytest' in sys.modules or 'test' in sys.argv:
+    env.read_env('.new_env.test')
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
